AppImageBuilder.test.Dockerfile

```diff
@@ -27,11 +27,22 @@ RUN <<EOR
     fi
   }
 
+  prepare() {
+    if command -v apt-get >/dev/null; then
+      apt-get update && apt-get install -y --no-install-recommends libx11-6
+    elif command -v pacman >/dev/null; then
+      pacman -Sy --noconfirm libx11
```

Comment:
This can break severely, we should either -S if the mirror files are already there and we're not too out of date as for the mirrors not to carry the package anymore, or -Syu to update the entire system to ensure things won't be linked against old libraries

https://wiki.archlinux.org/title/System_maintenance#Partial_upgrades_are_unsupported

Reply:
I'd vote for `-S` then because the base image is set to `latest` so that upgrades shouldn't be required.

Reply:
`-S` [does not work](https://github.com/git-developer/sc-controller/actions/runs/10972288457/job/30468554778), I changed it to `-Syu`.

Reply:
Looks like the image indeed does not carry mirror files to save on filesize, so we can't -S, shame, but it shouldn't matter to -Syu as with `latest` it will either be fully up to date or almost up to date, so it's not like we're pulling a two gigs to pull a dep.